js/src/common/components/Navigation.js

```diff
@@ -53,6 +53,7 @@ export default class Navigation extends Component {
         e.preventDefault();
         history.back();
       },
+      'aria-label': app.translator.trans('core.lib.nav.back'),
```

Comment:
It'd be better to replace the title parameter above with `aria-label` instead, `previous.title` is already being implicitly used right now as `aria-label` and is more descriptive.